litmus-portal/graphql-server/manifests/cluster-subscriber.yml

```diff
@@ -405,6 +405,10 @@ spec:
             - server
           image: argoproj/argocli:v2.9.3
           name: argo-server
+          resources:
```

Comment:
remove limits

Reply:
Actually these limits have been requested by @rajdas98 @ksatchit and @ajeshbaby ... we can increase the value here but its needed i guess.. @gdsoumya

Reply:
We haven't properly evaluated the resource requirements, these values might not work in production with multiple workflows and resources. And moreover a user might have different resource constraints, he can change it as required.